segmentation/include/pcl/segmentation/extract_polygonal_prism_data.h

```diff
@@ -145,7 +146,7 @@ namespace pcl
       }
 
       /** \brief Get the height limits (min/max) as set by the user. The
-        * default values are -FLT_MAX, FLT_MAX. 
+        * default values are std::numeric_limits<float>::lowest(), std::numeric_limits<float>::max().
```

Comment:
This shouldn't be here - its describing the fields, not the method?
Also height_limit_min_ is default 0, not lowest as the help text says?

Reply:
I've updated the lower limit. But it seems in filters/include/pcl/filters/passthrough.h and filters/include/pcl/filters/voxel_grid.h getFilterLimits, these kinds of comments are all put above the methods, should they all be updated?

Reply:
Hmm, what do you think @mvieth ? Should we just keep comments as is?

Reply:
I don't mind if the documentation mentions the default values